MSAL/src/native_auth/public/state_machine/state/SignInStates.swift

```diff
@@ -47,18 +48,18 @@ import Foundation
         scopes: [String],
         controller: MSALNativeAuthSignInControlling,
         inputValidator: MSALNativeAuthInputValidating = MSALNativeAuthInputValidator(),
-        flowToken: String) {
+        flowToken: String,
+        correlationId: UUID) {
         self.scopes = scopes
-        super.init(controller: controller, inputValidator: inputValidator, flowToken: flowToken)
+            super.init(controller: controller, inputValidator: inputValidator, flowToken: flowToken, correlationId: correlationId)
```

Comment:
nit: two tabs at start of line of code, please check Swiftlint

Reply:
Done.